docs/modules/ROOT/pages/ios_faq.adoc

```diff
@@ -0,0 +1,49 @@
+= iOS Frequently Asked Questions (FAQ)
+:hardbreaks:
+:oauth2-app-url: https://marketplace.owncloud.com/apps/oauth2
+:toc: right
+
+Here you can find some of the most frequently asked questions about the ownCloud iOS app.
+
+== Usage
+
+=== Compatibility with ownCloud server older than version 10.0
+
+Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Compatibility with iOS versions older than iOS 12.0
+
+Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
```

Comment:
pls use a macro (like the :oauth2-app-url:) for the link

Reply:
fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328